dev-client/src/screens/SettingsScreen/components/VersionIndicatorComponent.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useTranslation} from 'react-i18next';
+import {Text} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {APP_CONFIG} from 'terraso-mobile-client/config';
+
+export function VersionIndicator() {
+  const {t} = useTranslation();
+
+  return (
+    <Text variant="body2">
+      {APP_CONFIG.version
+        ? t('settings.version', {version: APP_CONFIG.version})
+        : `(${t('settings.unknownVersion')})`}
```

Comment:
```suggestion
        : `(${t('settings.unknown_version')})`}
```

Reply:
I have created https://github.com/techmatters/terraso-mobile-client/issues/1258 to address this issue at some point in the future. The instances introduced in this PR are now fixed.